posts/embracing_startup_agility_in_retail.md

```diff
@@ -0,0 +1,61 @@
+
+The retail industry has been squeezed on many fronts over the past decade, from Internet giants flexing their technology muscle (and willingness to not always turn a profit) to disrupt markets, to fast-moving startups who are unincumbered by big company process, and who are able to attract strong technology talent.
+
+As companies grow, it is an almost enevitable rite of passage that stricter processes are embedded, decision making can become more centralised, and increasing layers added to the heirarchy. While in some measure these things can help an organisation to scale, they can significantly reduce an organisations agility when it comes to embracing changing commercial landscapes.
+
+Established retailers can take conscious steps toward adopting and embracing startup levels of agility, ideally without the relentless late nights.
+
+We've observed 6 ways retailers can embrace startup agility:
+
+
+## 1. Empower and incentivise teams
+
+Particularly in large organisations, individuals and teams may not be fully empowered to deliver their program of work. Teams that are reliant on outside stakeholders for decisions or for additional support typically show a much lower level of performance than a team that is empowered and incentivised for end-to-end delivery.
+
+Often this way of working involves pulling together cross functional teams - people from different areas of the business who are committed to delivering the same common goal.
+
+
+## 2. Hire intrapreneurial people
+
+When hiring, targeting people who have an entrepreneurial streak can be a real benefit. People who are not afraid to challenge the ideas of their seniors, and who are proactively looking to improve themselves can inject some additional energy in to a team.
+
+Examining the profile of a typical early-stage startup hire, you'd likely be looking at someone who wears many hats - a bit of marketing, a bit of operations, and hopefully a whole load of hussle. Even if they've never done it before, someone who's going to figure out what needs to be done to make important things happen.
+
+
+## 3. Reward innovation
+
+Many established businesses are focused heavily on risk mitigation, with individuals rewarded for following safe strategies rather than challenging and innovating.
+
+Where desirable, we see benefit in incentivising individuals and teams to take risks, to test new markets, and to help the organisation learn from this risk taking.
+
+An organisation that primarily rewards people based on taking the safe, tried and tested path needs to be confident that they're not at risk from outside disruption.
+
+
+## 4. Adopt a test-and-learn strategy
+
+Most retailers have long since built out a data capability. Extending this a step further, startups often approach their entire business model with a test-and-learn strategy:
+
+Come up with a hypothesis, figure out the smallest experiment that will prove this, run the experiment, and repeat.
```

Comment:
Given it's preceded by a colon and is a new paragraph, feels like this should be a bullet point list.